app/templates/descriptor/confirm_resources.html

```diff
@@ -42,16 +42,12 @@ <h3 class="ui red block header">
                         <thead>
```

Comment:
in line 24 where we mention which option descriptor value, i think we should emphasize it more since it seems hard to read:
<img width="876" alt="screen shot 2017-01-14 at 11 04 52 pm" src="https://cloud.githubusercontent.com/assets/5404536/21960158/e6e5fdc4-daad-11e6-893e-6b4fec9d964c.png">

maybe putting bold `<b></b>` tags around the option value name?